src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/CarouselViewUITests.LoopNoFreeze.cs

```diff
@@ -22,14 +22,9 @@ public CarouselViewLoopNoFreeze(TestDevice device)
 		[Test]
 		[Category(UITestCategories.CarouselView)]
 		[Ignore("Currently fails")]
```

Comment:
Is this one still true? 

This test seems a bit weird, we have the Ignore attribute already but then also ignore it from within the method?

Can we see if the test runs OK on Android, macOS and Windows and then remove this Ignore attribute and only ignore it for iOS?